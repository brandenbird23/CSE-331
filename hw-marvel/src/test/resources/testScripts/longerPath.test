# Tests to find a longer path (via discussion post)

LoadGraph marvel marvel.csv

FindPath marvel SACRIFICER SUMMER-KLEIN