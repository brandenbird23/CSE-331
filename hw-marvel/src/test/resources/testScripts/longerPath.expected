# Tests to find a longer path (via discussion post)

loaded graph marvel

path from SACRIFICER to SUMMER-KLEIN:
SACRIFICER to HAWK via AA2-28
HAWK to BARON-STRUCKER/WOLFG via ST-156
BARON-STRUCKER/WOLFG to HANNIBAL/HENRY-KING via NS-2
HANNIBAL/HENRY-KING to SUMMER-KLEIN via JIM-520